blog-posts/scoped-registry.md

```diff
@@ -0,0 +1,128 @@
+---
+path: "/blog/scoped-registry/"
+date: "2021-12-30"
+title: "How to set up multiple npm registries with scoped registries"
+featuredImage: images/npm-scoped-registry.jpg
+attribution:
+    creator: Paul Esch-Laurent
+    source: https://unsplash.com/photos/oZMUrWFHOB4
+seoMetaText: How to set up multiple npm registries with scoped packages and scoped registries even if they are protected
```

Comment:
This need would need some more love depending on your decision to change the title or not.

Reply:
Updated the texts :+1: I think they are now way better.